src/pages/Home/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Layout from '../../components/Layout/Layout';
+// eslint-disable-next-line import/extensions
+import CarsList from '../../components/CarsList/CarsList';
```

Comment:
- [ ] The homepage won't load on my end:

Browser:

![image](https://github.com/bernabasy/ride-royale-front-end/assets/57421823/47b5e8d4-1d51-4c66-8a2f-926b2ce5081f)

Console:

![image](https://github.com/bernabasy/ride-royale-front-end/assets/57421823/9b907d62-8f01-4b20-8b41-ceaa71a0dd8c)

## Tips and extra help

This is because you have 2 components folders:  `Components` and `components`:

![image](https://github.com/bernabasy/ride-royale-front-end/assets/57421823/59a0bebd-3b94-4645-b137-fa605676d484)

The one that is capitalized is the one that has the data for the import:

![image](https://github.com/bernabasy/ride-royale-front-end/assets/57421823/73d5f7c8-e042-47a1-8264-9af373ce4f2d)

I was able to make this work by correcting the import with the correct `Components` folder. 

```javascript
import CarsList from '../../Components/CarsList/CarsList';
```


Kindly correct it. 💪